infra/examples-dev/aws/variables.tf

```
@@ -121,6 +121,12 @@ variable "force_bundle" {
   default     = false
 }
 
+variable "enable_deployment_testing" {
```

```suggestion
variable "provision_testing_infrastructure" {
```
I find the name a bit confusing:
An `enable_deployment_testing = false` looks like you don't want to do testing, rather than infra is not needed.